packages/xstate-react/src/useMachine.ts

```diff
@@ -27,17 +25,13 @@ const isEqual = (prevState: AnyState, nextState: AnyState) => {
 
 export interface UseMachineOptions<
   TContext extends MachineContext,
-  TEvent extends EventObject
+  _TEvent extends EventObject
 > {
   /**
    * If provided, will be merged with machine's `context`.
    */
+  // TODO: deprecate & remove context as option
   context?: Partial<TContext>;
-  /**
-   * The state to rehydrate the machine to. The machine will
-   * start at this state instead of its `initialState`.
-   */
-  state?: StateConfig<TContext, TEvent>;
```

Comment:
I feel like this had much better DX - perhaps the option should be renamed though (`persistedState`?). Having to call `.at` (or any other method used for this) should be hidden from the consuming sites.

I feel like this is a similar story to `input`. The rehydrated state is the `interpret`'s argument and it's Interpreter's job to correctly resolve it internally through ActorBehavior's APIs. And since `useMachine`/`useInterpret` correlates closely to the `interpret` call - I feel that providing this argument to both of them should look very similar. And it does look similar now - it's just that I propose making it the `interpret`'s argument.

Reply:
It's especially important in React since we shouldn't pass new machines to the `useMachine` on each render and that's exactly what happens with code like this:
```ts
useMachine(machine.at(state))
```